std/markdown.joke

```diff
@@ -4,7 +4,14 @@
   markdown)
 
 (defn ^String convert-string
-  "Returns the HTML rendering of Markdown string s"
+  "Returns the HTML rendering of Markdown string s `opts` is an optional map of boolean rendering options (all default to tru to `true`)
```

Comment:
Missing `.` after `string s`. Also typo: `to tru`. Finally, there is no markdown support in docstrings, so "`" should not be used. So something like:

> "Returns the HTML rendering of Markdown string s. opts is an optional map of boolean rendering options (all default to true)"